packages/wonder-blocks-typography/src/__docs__/typography.argtypes.js

```diff
@@ -0,0 +1,31 @@
+export default {
+    children: {
+        control: {type: "text"},
+        description: "Text to appear with the specified typography styles.",
+        table: {type: {summary: "React.Node"}},
+        type: {required: false},
+    },
+    style: {
+        control: {type: "object"},
+        description: "Optional custom styles.",
+        table: {type: {summary: "StyleType"}},
+        type: {required: false},
+    },
+    testId: {
+        control: {type: "text"},
+        description: "Test ID used for e2e testing.",
+        table: {type: {summary: "string"}},
+        type: {required: false},
+    },
+    tabIndex: {
+        control: {type: "number"},
+        table: {type: {summary: "number"}},
+        type: {required: false},
+    },
```

Comment:
**sugestion:** Even that the types use `tabIndex`, I think it would be better to not include this in the documentation as it might mislead devs about using this prop for headings? wdyt? do you think there would still be cases where we should/need to use `tabIndex` for headings and/or paragraphs?

Reply:
I agree with you. I was about to ask why that's even there, but `TextViewSharedProps` is supposed to be pretty versatile so I guess it makes sense.